workspace.go

```diff
@@ -100,6 +107,37 @@ func (w *Workspace) evalCommand(command string) (EvalResult, error) {
 	return res, err
 }
 
+func (w *Workspace) BuildCommit(cmd string) (EvalResult, error) {
+	res, err := w.build(cmd)
+	w.CurrentImage = res.NewImage
+	w.history = append(w.history, res)
+	return res, err
+}
+
+func (w *Workspace) Build(cmd string) (EvalResult, error) {
+	res, err := w.build(cmd)
+	res.Deleted = true
+	w.history = append(w.history, res)
+	return res, err
+}
+
+func (w *Workspace) build(cmd string) (EvalResult, error) {
+	name := uuid.New()
+	c := fmt.Sprintf("FROM %s\n%s", w.CurrentImage, cmd)
+	f, _ := os.Create("." + name)
```

Comment:
Should we put tmp files into a .cyclops directory for containment?

Are there any error cases where the temp file doesn't get deleted?  What happens if someone control-c's cyclops during the build?